src/XliffTasks/build/XliffTasks.targets

```diff
@@ -126,7 +126,7 @@
   <Target Name="TranslateSourceFromXlf"
           Condition="'$(EnableXlfLocalization)' == 'true'"
           DependsOnTargets="GatherXlf;BatchTranslateSourceFromXlf"
-          BeforeTargets="PrepareResourceNames;$(PrepareResourceNamesDependsOn)"
+          BeforeTargets="VSCTCompile;PrepareResourceNames;$(PrepareResourceNamesDependsOn)"
```

Comment:
This seems right, but now I'm not sure how it's working for other projects. :( Is there anything unique about your project that would cause VSCTCompile to run after PrepareResourceNames?

Reply:
Not that I can see, unless this is fallout from my previous PR that gathered translatable items at a different point, so maybe the collection was already updated?

Reply:
Could it be this: https://github.com/Microsoft/visualfsharp/blob/46a1e68addb8c1f685bc661b129758f2d047d237/vsintegration/src/FSharp.ProjectSystem.FSharp/ProjectSystem.fsproj#L37

I've got some testing to do to make sure other repos aren't affected?